src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+@mixin catalog-grid($columns) {
+  display: grid;
+  grid-template-columns: repeat($columns, $card-width);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+.cards {
+  padding-top: 50px;
+  padding-bottom: 50px;
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    @include catalog-grid(2);
+  }
+
+  @media (min-width: 768px) {
+    @include catalog-grid(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include catalog-grid(4);
+  }
+}
+
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)